ansible/files/runtimes.json

```diff
@@ -128,6 +128,20 @@
                     "attachmentName": "codefile",
                     "attachmentType": "text/plain"
                 }
+            },
+            {
+                "kind": "swift:5.3",
+                "default": false,
```

Comment:
intended to leave as not the default?

Reply:
wasn't sure what the right thing to do was.  Jumping from 4.2 to 5.3 as the default seemed aggressive, but maybe we should.

Reply:
I defer to your judgement.

Reply:
decided to go for making 5.3 as default.  Will update PR title.